app/controllers/api/events_controller.rb

```diff
@@ -0,0 +1,57 @@
+# rubocop:disable Style/ClassAndModuleChildren
+# :reek:InstanceVariableAssumption
+
+class Api::EventsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def index
+    @events = Event.all
+    render json: @events
+  end
+
+  def show
+    @event = Event.find_by(id: params[:id])
```

Comment:
you are using such line in several places. it makes sense to move it in `before_action`